### PR TITLE
Add release notes for 1.13.1

### PR DESCRIPTION
**Pull request by morphis**

### CODE REVIEW

File: announcements/1.13.1.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+### Introduction
+
+The Anbox Cloud team is pleased to announce the release of Anbox Cloud 1.13.1.
+
+Please see [component versions](https://anbox-cloud.io/docs/component-versions) for a list of updated components.
+
+#### New features & improvements
+
+ * Android security updates for January 2022 (see [Android Security Bulletin - March 2022](https://source.android.com/security/bulletin/2022-03-01) for more information)
```
Comment (ru-fu) [inline announcements/1.13.1.md:9]:
```suggestion
 * Android security updates for March 2022 (see [Android Security Bulletin - March 2022](https://source.android.com/security/bulletin/2022-03-01) for more information)
```

File: announcements/1.13.1.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+ * Android WebView has been updated to [99.0.4844.58](https://chromereleases.googleblog.com/2022/03/stable-channel-update-for-desktop.html)
```
Comment (ru-fu) [inline announcements/1.13.1.md:10]: The link says 99.0.4844.51, but we might be using a newer version?

File: release-notes.md [hunk 3]
```diff
@@ -2,6 +2,40 @@
 
 See [Upgrade Anbox Cloud](https://discourse.ubuntu.com/t/upgrading-from-previous-versions/17750) or [Upgrade the Anbox Cloud Appliance](https://discourse.ubuntu.com/t/upgrade-anbox-cloud-appliance/24186) for instructions on how to update your Anbox Cloud deployment.
 
+[Details=1.13.1]
+
+## 1.13.1 (March 21 2022)
+
+### New features & improvements
+
+ * Android security updates for January 2022 (see [Android Security Bulletin - March 2022](https://source.android.com/security/bulletin/2022-03-01) for more information)
```
Comment (ru-fu) [inline release-notes.md:11]:
```suggestion
 * Android security updates for March 2022 (see [Android Security Bulletin - March 2022](https://source.android.com/security/bulletin/2022-03-01) for more information)
```

**Review by ru-fu — CHANGES_REQUESTED** [review]

File: announcements/1.13.1.md (hunk 2 above)
Comment (morphis) [reply announcements/1.13.1.md:10]: There is no blog post for .58 but .51 and .58 are from the same build, just different revs for different platforms (Desktop, Android).

**Pull request merged**